src/main/frontend/db/model.cljs

```diff
@@ -1094,6 +1094,47 @@
       (util/safe-page-name-sanity-lc page))
      (distinct))))
 
+; return list of parents namespace
```

Comment:
Great to see docs for your fns. Would be good to associate this with a fn. See `get-paginated-blocks-no-cache` in this file for an example